config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_iapsettings.iap.cnrm.cloud.google.com.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     plural: iapsettings
     shortNames:
     - gcpiapsettings
-    - gcpiapsettingss
```

Comment:
I thinkwe may wnat to use `gcpiapsetting` and `gcpiapsettings` as short name. And change the singular to `iapsetting` (but this would be a breaking change)